### PR TITLE
Get insight legend toggling on Pie charts working

### LINKED ISSUE #8236: Legend toggling not working for pie charts
## Bug description

Toggling keys in legend and table doesn't work for pie chart.

## Expected behavior

Toggling legend/table key checkboxes should show/hide corresponding sections of pie.

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Fixes #8236

https://user-images.githubusercontent.com/13460330/151078453-a4960c9b-537e-4275-9021-e6ddae9b5868.mov

## How did you test this code?

Manual QA

### CODE REVIEW

File: frontend/src/lib/experimental/FeedbackCallCTA.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ const feedbackCallLogic = kea<feedbackCallLogicType>({
     events: ({ actions, values, cache }) => ({
         afterMount: () => {
             if (
-                values.featureFlagGroup?.startsWith('variant') &&
+                values.featureFlagGroup?.startsWith?.('variant') &&
```
Comment (alexkim205) [inline frontend/src/lib/experimental/FeedbackCallCTA.tsx:77]: flyby bug fix that kept breaking dev env

**Review by rcmarron — APPROVED** [review]
LGTM

File: frontend/src/lib/experimental/FeedbackCallCTA.tsx (hunk 1 above)
Comment (rcmarron) [reply frontend/src/lib/experimental/FeedbackCallCTA.tsx:77]: 🙏 I was just hitting the same thing. I think it was the `sync_feature_flag` command setting a feature flag to false that was supposed to be a mutli-variant.

Comment (neilkakkar) [reply frontend/src/lib/experimental/FeedbackCallCTA.tsx:77]: oh, interesting, makes sense.

**Pull request merged**